### PR TITLE
DPU l-filesToParliament initial version

### PR DESCRIPTION
**Pull request by mkl-eea-sk**
Feature/l files to parliament

### CODE REVIEW

**Comment by mkl-eea-sk** [comment]
resolves #171

File: l-filesToParliament/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# L-FilesUpload #
```
Comment (skrchnavy) [inline l-filesToParliament/README.md]: update Readme.md for L-FilesToParliament

File: l-filesToParliament/src/main/resources/about.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Can be used as replacement for DPUs:</br>
```
Comment (skrchnavy) [inline l-filesToParliament/src/main/resources/about.html]: To remve or to convert to L-FilesToParliament about

File: l-filesToParliament/src/main/java/eu/unifiedviews/plugins/loader/filestoparliament/FilesToParliament.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                    .addTextBody("dataFormat", rdfFormat, ContentType.MULTIPART_FORM_DATA)
+                    .addBinaryBody("statements", new File(URI.create(entry.getFileURIString())), ContentType.DEFAULT_BINARY, filename);
+
+            HttpEntity entity = entityBuilder.build();
+            httpPost.setEntity(entity);
+            response = client.execute(httpPost);
+
+            if (response.getStatusLine().getStatusCode() != 200) {
+                throw ContextUtils.dpuException(ctx, "FilesToParliament.execute.uploadFail");
+            }
```
Comment (tomas-knap) [inline l-filesToParliament/src/main/java/eu/unifiedviews/plugins/loader/filestoparliament/FilesToParliament.java]: Please provide to the user the response code from the server, not just "It failed"

File: l-filesToParliament/src/main/java/eu/unifiedviews/plugins/loader/filestoparliament/FilesToParliament.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    protected void innerExecute() throws DPUException {
+        CloseableHttpClient client = null;
+        try {
+            client = HttpClients.createDefault();
+            for (FilesDataUnit.Entry entry : FilesHelper.getFiles(filesInput)) {
+                String filename = VirtualPathHelpers.getVirtualPath(filesInput, entry.getSymbolicName());
+                if (StringUtils.isEmpty(filename)) {
+                    filename = entry.getSymbolicName();
+                }
+                sendFile(ctx, config.getBulkUploadEndpointURL(), client, config.getRdfFileFormat().toUpperCase(), filename, entry);
```
Comment (tomas-knap) [inline l-filesToParliament/src/main/java/eu/unifiedviews/plugins/loader/filestoparliament/FilesToParliament.java]: If everything went ok, there should be log, at least on debug level, stating that File X was uploaded successfully.

**Comment by tomas-knap** [comment]
Also there is a feature request to be able to specify named graph - see: https://github.com/UnifiedViews/Plugins/issues/171

**Comment by mkl-eea-sk** [comment]
#171 is closed, could this be merged?

**Pull request merged**